rosdep/python.yaml

```diff
@@ -6863,6 +6863,9 @@ python3-pyside2.qtopengl:
     '*': [python3-pyside2.qtopengl]
     bionic: null
     xenial: null
+python3-pyside2.qtuitools:
+  debian: [python3-pyside2.qtuitools]
+  ubuntu: [python3-pyside2.qtuitools]
```

Comment:
Since this package is only available since `focal`, we need to change this entry a bit.  It should look like:

```
ubuntu:
  '*': [python3-pyside2.qtuitools]
  bionic: null
  xenial: null
```

Reply:
I've adjusted the availability in Ubuntu as you suggested.

Reply:
Thanks for iterating, looks good.